### PR TITLE
[Frontend] Implement Broadcast page Design #53

### PR DESCRIPTION
**Pull request by kenkirito**
## Broadcast page

 Closes: # (issue number)

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

HITK Tech Community and 9 more pages - Personal - Microsoft​ Edge 06-01-2021 01_39_41 AM
Comparing HITK-TECH-Community_main kenkirito_main · HITK-TECH-Community_Community-Website and 11 more pages - Personal - Microsoft​ Edge 06-01-2021 03_49_10 AM (1)

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: frontend/src/components/broadcast/broadcast_second.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+export class Owldemo1 extends Component {
+    render() {
+
+
+    return (
+
+        <div className="Bottom_bgbox">
+        <div className="Bottom_bgbox_first">
+      <div className="motive">
+          <h1 style={{ color: "#1B2431" }}>Previsely Broadcast</h1>
```
Comment (salil-naik) [inline frontend/src/components/broadcast/broadcast_second.js:20]: Check the spelling error.

**Review by salil-naik — CHANGES_REQUESTED** [review]
Hi. Good job! Kindly do the following changes. 

- [ ] Make the page responsive. Should work well on tablet and mobile. 
- [ ] Give an opacity of 0.42 to the background of the modal. The content below the modal should be slightly visible as per the design. 
- [ ] Format all the files using prettier. 

- [ ] Mention which issue it closes.

File: frontend/src/components/broadcast/broadcast_second.js (hunk 1 above)
Comment (kenkirito) [reply frontend/src/components/broadcast/broadcast_second.js:20]: but in UI design, it was given completely white when the popout open

Comment (salil-naik) [reply frontend/src/components/broadcast/broadcast_second.js:20]: Yes, I'm aware. Giving an opacity will make it look better. 😃 Anything that enhances the UI is appreciated.

Comment (kenkirito) [reply frontend/src/components/broadcast/broadcast_second.js:20]: ok

**Comment by kenkirito** [comment]
HITK Tech Community and 8 more pages - Personal - Microsoft​ Edge 06-01-2021 01_31_53 AM

**Comment by kenkirito** [comment]
@salil-naik  is dialog background fine

**Comment by salil-naik** [comment]
> @salil-naik  is dialog background fine
> 

Yes. Looks good.

**Comment by kenkirito** [comment]
thanks

**Comment by kenkirito** [comment]
HITK Tech Community and 18 more pages - Personal - Microsoft​ Edge 06-01-2021 07_13_34 PM
HITK Tech Community and 18 more pages - Personal - Microsoft​ Edge 06-01-2021 07_13_27 PM (1)

**Comment by salil-naik** [comment]
@kenkirito It works fine on a tablet but breaks on a mobile device. Please make it responsive for mobile too. And your owl-carousel css code breaks **our motive** section in the homepage. I have commented on the line of code which you need to change. Kindly fix that.

File: frontend/src/components/broadcast/owl.css [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  text-decoration: none;
+  &:hover {
+    color: black;
+  }
+}
+
+/* For mobile phones: */
+[class*="col-"] {
+  width: 100%;
+}
```
Comment (salil-naik) [inline frontend/src/components/broadcast/owl.css:131]: This piece of code will affect all the **bootstrap col** classes across the pages. Please remove it. 

You may refer to [this](http://owlcarousel2.github.io/OwlCarousel2/demos/responsive.html) to make owl carousel responsive.

**Review by salil-naik — CHANGES_REQUESTED** [review]

File: frontend/src/components/broadcast/owl.css (hunk 2 above)
Comment (kenkirito) [reply frontend/src/components/broadcast/owl.css:131]: ok

Comment (kenkirito) [reply frontend/src/components/broadcast/owl.css:131]: can you tell me the dimensions of mobile view??

**Comment by kenkirito** [comment]
DONE

**Comment by kenkirito** [comment]
please check

File: frontend/src/components/broadcast/owl.css (hunk 2 above)
Comment (salil-naik) [reply frontend/src/components/broadcast/owl.css]: You will get the dimensions in DevTools.

Comment (kenkirito) [reply frontend/src/components/broadcast/owl.css]: I am done please check my pr

**Comment by salil-naik** [comment]
@kenkirito It's not yet fixed. Check the images I have attached. 

The card is not visible in mobile view. You can reduce the padding on either side and give more area to the card. 
Also, fix the spelling error. **Previously**
image

Both in mobile and tablet view, the cards are coming above the menu. Reduce the z-index of the cards. 
image

**Comment by kenkirito** [comment]
i have fixed the spelling error

**Comment by kenkirito** [comment]
and I haven't used any z index

**Comment by kenkirito** [comment]
I will try to resolve the issue

**Comment by kenkirito** [comment]
bro please merge my request i am done with the responsiveness

**Comment by kenkirito** [comment]
almost done with the menu request

**Comment by kenkirito** [comment]
HITK Tech Community and 12 more pages - Personal - Microsoft​ Edge 07-01-2021 10_28_06 PM

**Comment by kenkirito** [comment]
https://user-images.githubusercontent.com/68530218/103926098-7c4a5300-513e-11eb-964f-be776aec8c1a.mp4

**Comment by kenkirito** [comment]
@salil-naik please see the video i have done all the changes

**Comment by kenkirito** [comment]
@salil-naik  please review my changes

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @kenkirito Thanks for incorporating all the change spatiently. It almost ready to go, just make these changes
1. I can see the change in `navbar.css` can you please revert that.
2. For small screens, increase the card width.It would look much better that way.

**Comment by kenkirito** [comment]
i have done chnages in navbar because .owl-carousel class has a z-index of 1, where as the menu itself is also 1. Relative leements have a precedence in the stack with a same z-index, so just setting the menu z-index to something higher, like preferably over 100 is your solution.

**Comment by kenkirito** [comment]
i have done the changes

**Comment by kenkirito** [comment]
HITK Tech Community and 2 more pages - Personal - Microsoft​ Edge 08-01-2021 01_38_22 PM (1)

**Comment by kenkirito** [comment]
https://lakshya-gmail-3.tinytake.com/media/ef8dc6?filename=1610111647660_TinyTake_08-01-2021-06-43-55.png&sub_type=thumbnail_preview&type=attachment&width=1199&height=760

**Review by Kajol-Kumari — APPROVED** [review]
Hey @kenkirito I really appreciate your patience and hard work but it's still not up to the mark. Maybe we should open smaller issues as this seems quite big. We are merging this as of now.

Happy learning :)

**Pull request merged**